packages/clusview/pipelines/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Functions for pipelines for metric mapping
```

Comment:
Although right now this only has a metric mapping pipeline, in the future this package will also accommodate other types of pipelines, not only metric mapping necessarily. So, simply describing that this includes pipelines in the form of chaining functionality and components would suffice.